apps/web/nuxt.config.ts

```diff
@@ -63,6 +63,7 @@ export default defineNuxtConfig({
       noCache: process.env.NO_CACHE || '',
       configId: process.env.CONFIG_ID || '',
       isHero: process.env.IS_HERO === 'true',
+      defaultFont: process.env.TEXT || 'Red Hat Text',
```

Comment:
```suggestion
      font: 'Red Hat Text',
```
We should move away from referring to custom environment variables in this context. It'll be possible change the configuration via e.g. `NUXT_PUBLIC_FONT` and do so without rebuilding the app.